tests/snippets/elpi/test_pred.txt

```diff
@@ -0,0 +1,61 @@
+---input---
+pred p1.
+pred p2 i:int, o:list A.
+pred p3 i:(bool -> prop).
+:index(_ 2) pred p4 i:int, i:A.
+
+---tokens---
+'pred'        Keyword.Declaration
+' '           Text
+'p1'          Name.Function
+'.'           Text
+'\n'          Text
+
+'pred'        Keyword.Declaration
+' '           Text
+'p2'          Name.Function
+' '           Text
+'i:'          Keyword.Mode
+'int'         Keyword.Type
+','           Text
+' '           Text
+'o:'          Keyword.Mode
+'list'        Keyword.Type
+' '           Text
+'A'           Keyword.Type
+'.'           Text
+'\n'          Text
+
+'pred'        Keyword.Declaration
+' '           Text
+'p3'          Name.Function
+' '           Text
+'i:'          Keyword.Mode
+'('           Keyword.Type
+'bool'        Keyword.Type
+' '           Text
+'-'           Error
```

Comment:
This is missing some rule, we don't allow error tokens.

Reply:
the discussion seems to say that this is OK

Reply:
Is this an error in the source language? I assume this is a valid piece of code and in that case Error is not OK. I'd also prefer error cases to be in a separate file instead of mixed with the rest because they are exceptional.